README.md

```diff
@@ -1,103 +1,124 @@
 # C++ example module
 
-This is our example module for C++. It shows you how to:
+This example module demonstrates the use of the [C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk) to build a minimal Viam module.
+It is [available for use on the registry](https://app.viam.com/module/viam/module-example-cpp) and shows how to include CI (Continuous Integration) builds for both x86 and ARM architectures.
 
-- Build a minimal Viam module using our [C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk)
-- With CI builds for x86 and ARM
-- Released through our registry
+You can use this as a reference along with the C++ instructions for [coding your own modules](https://docs.viam.com/registry/create/).
 
-## Local build environment
 
-Known good environments:
-- Debian Bullseye (this is what our CI uses)
-- You can use Ubuntu 22.04 for testing, but the C++ builds will not be binary compatible with Bullseye
+## Requirements
 
-Setup and build instructions:
-- Install the apt packages in [apt-setup.sh](apt-setup.sh)
-- Install cmake
-	- Check CMAKE_VERSION in the github workflow (3.27.4 as of this writing)
-	- On Ubuntu 22.04, `snap install cmake` is new enough but `apt install cmake` is not
-	- Otherwise follow [CMake's download instructions](https://cmake.org/download/)
-- [Install rust](https://www.rust-lang.org/tools/install) (needed for viam_rust_utils)
+Before building this module, verify your operating system's development environment is compatible.
+The tested ideal environment for building this module is Debian Bullseye (used in our CI).
 
-## Deploy to robot
+You can also use Ubuntu 22.04 for testing, but the resulting C++ builds will not be binary compatible with Bullseye.
 
-These instructions assume you're targeting something similar to Debian bullseye.
 
-First install libraries on your target system:
+To setup for building this module:
+
+1. Install the apt packages in [apt-setup.sh](apt-setup.sh).
```

Comment:
(q) We've got these `apt install` instructions here but below (at lines 29-31) are another set of instructions. Can these be combined with a single simple command, e.g. `sh ./apt-setup.sh`?

Reply:
If there's no reason to keep these separate, then yes it looks like we could just add the two additional packages to `apt-setup.sh`. I've made a PR doing so if this is the correct thing to do: https://github.com/viamrobotics/module-example-cpp/pull/5

Then these docs could indeed just suggest `sh ./apt-setup.sh` for the one-stop shop of dep resolution, both for initial set up and for ongoing usage with the GitHub action.